extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -265,7 +265,7 @@ function arrayindex( &$parser, $arrayid , $index , $options='') {
 *
 *  See: http://www.php.net/manual/en/function.count.php
 */
-    function arraysize( &$parser, $arrayid) {
+    function arraysize( $parser, $arrayid) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)